scripts/sidebar/components/icon/giphy/style.scss

```diff
@@ -0,0 +1,10 @@
+.dropit-giphy-sidebar-icon,
```

Comment:
Minor: Not a big fan of the directory structure, I prefer more something like `components/{unspalsh/giphy}/{icon/sidebar/menu-item}/index.js`

Reply:
hum to me Giphy Unsplash are API not really components but I'm really not an expert in js architecture so it can be changed easily I think.